woocommerce/woocommerce-android
# Edit Products Release 1 - Update Product Inventory - Step 3

This PR addresses step 3 of #1614 by adding the option to verify if a SKU exists for a site when user edits the inventory settings. Note that this PR only adds the option to verify if SKU already exists for a site. The logic to actually update the product inventory will take place in another PR.

#### Notes
- Targeting develop branch since this change affects only debug users.
- In addition to verifying if SKU exists, this PR also adds the option to display a discard dialog if there are any changes made to the product inventory fields (when back is clicked).
- The `DONE` menu button is only displayed if any one of the product inventory fields have been updated. It is greyed out, if the SKU entered, already exists for a site and error is displayed.

#### Screenshots
<img width="300" src="https://user-images.githubusercontent.com/22608780/73424268-57593780-4354-11ea-8eb5-11e88aadc884.gif">

#### Testing
- Verify that if you enter a SKU that already exists for a site, an error message is displayed under the SKU field and the DONE button is greyed out.
- Verify that if any fields are edited in the product inventory screen and back is clicked, the discard dialog is displayed.


Update release notes:
